cmd/buildah/images_test.go

```diff
@@ -506,15 +506,22 @@ func TestMatchesSinceImageFalse(t *testing.T) {
 
 func captureOutputWithError(f func() error) (string, error) {
 	old := os.Stdout
-	r, w, _ := os.Pipe()
+	r, w, err := os.Pipe()
+	if err != nil {
+		return "", err
+	}
 	os.Stdout = w
 
-	err := f()
+	if err := f(); err != nil {
+		return "", err
+	}
 
 	w.Close()
 	os.Stdout = old
 	var buf bytes.Buffer
-	io.Copy(&buf, r)
+	if _, err := io.Copy(&buf, r); err != nil {
+		return "", err
```

Comment:
Do we want to discard any received data here, for example if we get an "unexpected EOF" error?

Reply:
Let's play it safe and revert that. golangci-lint will now ignore this case.